src/Symfony/Bundle/SecurityBundle/Form/Type/UserLoginType.php

```diff
@@ -29,11 +28,11 @@
  */
 class UserLoginType extends AbstractType
```

Comment:
I would rather have this called a `UsernamePasswordType`, but tbh, I think this is something the developer should handle himself, because in the end, you don't do anything with the form as the security component handles it.